scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +658,32 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+    def apply(self, points):
+        """Apply this rotation on a set of points.
+
+        Rotates `points` by the rotation(s) represented in the object. In terms
```

Comment:
I think this is an appropriate place to explain "rotation-projection duality". That is we can either think of it as projection of a vector components from one frame to another, or physical rotation of a vector as if it is glued to the original frame as it rotates.